Services/CO.CDP.OrganisationInformation.Persistence/JoinRequest.cs

```diff
@@ -0,0 +1,21 @@
+using CO.CDP.EntityFrameworkCore.Timestamps;
+using Microsoft.EntityFrameworkCore;
+
+namespace CO.CDP.OrganisationInformation.Persistence;
+
+[Index(nameof(Guid), IsUnique = true)]
+public class JoinRequest : IEntityDate
```

Comment:
Suggestion, table name - OrganisationPersonAddRequest ?

Reply:
I still prefer join request. Had a session with @jakzal and we thought it appropriate. See https://lucid.app/lucidspark/cb2c1600-0bff-42f2-b5d1-d3c96321b0ca/edit?invitationId=inv_3c51f859-f04a-4885-aff3-a3d949d11acd&page=0_0# for more info

Reply:
Naming it `OrganisationJoinRequest` wouldn't hurt while giving more context?